src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -100,19 +105,29 @@
                                   (:library:datasetName data)))}
                      {:id "library:indication" :header (:title (:library:indication attributes))
                       :column-data :library:indication :initial-width 180}
-                     {:id "library:dataUseRestriction" :header (:title (:library:dataUseRestriction attributes))
-                      :column-data :library:dataUseRestriction :initial-width 180}
                      {:id "library:numSubjects" :header (:title (:library:numSubjects attributes))
-                      :column-data :library:numSubjects :initial-width 100}]
+                      :column-data :library:numSubjects :initial-width 100}
+                     {:id "library:consentCodes" :header (:title (:library:consentCodes attributes))
+                      :column-data :library:consentCodes :initial-width 180
+                      :as-text (fn [data] (string/join ", " (sort data)))
+                      :sort-by (comp string/lower-case :as-text)
```

Comment:
This isn't going to work, it will be trying to call `:as-text` on the data, which doesn't exist.